modules/end-user-guide/partials/proc_building-an-image-for-intellij-idea-community-or-pycharm-community.adoc

```diff
@@ -0,0 +1,99 @@
+[id="building-an-image-for-intellij-idea-community-or-pycharm-community_{context}"]
+= Building an image for IntelliJ IDEA Community or PyCharm Community
+
+This procedure describes how to build an image for IntelliJ IDEA Community or PyCharm Community.
+
+.Prerequisites
+
+* The build host has at least 2 GB of available RAM.
+
+* The build host has a local installation of following tools:
```

Comment:
`installation` = the act of putting a program onto a computer (source: dictionary.cambridge.org)

imho `local` (as in locally installed) is redundant and can be dropped if the location/host is specified (here `the build host`).

Reply:
fixed.